__tests__/app.test.js

```diff
@@ -419,4 +419,69 @@ describe("GET /api/users", () => {
       expect(msg).toBe('Not available')
     })
   })
+})
+describe.only("GET /api/articles with sorting queries", () => {
```

Comment:
This stops all other tests running. I can't review this as I don't know that all your other tests are passing.